features/tools.vitruv.testutils.changevisualization.feature/feature.properties

```diff
@@ -0,0 +1,7 @@
+featureName=Vitruv Test Utilities: Change Visualization
+description=Utilities for visualization changes on models
+providerName=vitruv.tools
+copyrightURL=http://vitruv.tools
+copyright=\
+Copyright (c) 2014-2022 Vitruv Tools Team\n\
+Karlsruhe Institute of Technology (KIT), Software Design and Quality, Karlsruhe, Germany
```

Comment:
We could update this to DSIS.

Reply:
True, but maybe we should have an issue and update all entries at the same time? Or we could even keep SDQ, because SDQ is still the "umbrella brand" of DSiS and MCSE.

Reply:
I have added this as a topic for or next ConCall.